beverage_distributor/demo/website_theme_apply.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <odoo>
-    <function name="make_scss_customization" model="web_editor.assets" context="{'website_id': ref('website.default_website')}">
-        <value eval="'/website/static/src/scss/options/colors/user_color_palette.scss'"/>
-        <value eval="{'o-color-1': '#5f4b8b','o-color-2': '#e69a8d','o-color-3': '#f2f0f5','o-color-4': '#FFFFFF','o-color-5': '#2e1d14','menu': 1}"/>
+    <function model="ir.module.module" name="_theme_load" context="{'apply_new_theme': True}">
```

Comment:
Did you forget <function name="button_immediate_install" model="ir.module.module" eval="[ref('base.theme_zap', raise_if_not_found=False)]"/> ?